src/word-utils.js

```diff
@@ -31,12 +31,24 @@ export function expandOneWordToAcronym(CapitalWord) {
  * @example World Health Organization -> WHO
  */
 export function expandWordsToAcronym(words) {
-    if (words.length === 1) {
+    if (words.length == 1) {
         return expandOneWordToAcronym(words[0]);
     }
+    else{
     // World Health Organization -> WHO
-    return words.reduce((acronym, word) => {
-        acronym += word.charAt(0);
-        return acronym;
-    }, "");
+        let result = [];
+
+        let output = words.reverse().reduce((acronym, word, i) => {
+            acronym.unshift(word.charAt(0))
+            result.push(acronym.join(""))
+            return acronym;
+        }, []);
+
+        let output_1 = words.reverse().reduce((acronyms, word, i) => {
```

Comment:
I do not understand what this code doing?

`acronyms` is array, but `acronyms += "string"` ???

Reply:
`acronyms` is `string`, `result` is my array of acronyms